### ISSUE TITLE
Add input validation on pages.

### ISSUE THREAD

**Issue by andwah** [Starcounter/Warehouse#44]
Input validation example:
![image](https://cloud.githubusercontent.com/assets/2656996/13375063/54f703d2-dd96-11e5-95cd-ee4b8d71f352.png)

**Comment by miyconst** [comment]
Would be cool, but not something really important. Anyway, we have to come up with some good validation pattern.

**Comment by andwah** [comment]
I dont agree, it's ***very*** important, this will help users to learn to use the system. 
We should help and guide the user as best as we can with feedback, constructive "error" messages, etc..

Crucial for a good UI.

**Comment by miyconst** [comment]
You misunderstood me. It is not important at this moment, but it is of course very important for good UI.

I just mean that we have to focus on functionality of Warehouse and WarehouseClient first and then on UI.

**Comment by andwah** [comment]
I agree,  It's in the later "Milestone 1.1 Beta"

**Comment by warpech** [comment]
Any progress with this? Me and @aloscha are also interested in the result.

Also, see https://github.com/StarcounterSamples/KitchenSink/issues/6

**Comment by andwah** [comment]
@warpech not at the moment. Still using the old way [PropertyMetadata](https://github.com/Smorgasbord-Development/PropertyMetadata).

**Comment by aloscha** [comment]
@andwah I've create simple pattenr for validation based on FluentValidator.
Available in KitchenSink on master - https://github.com/StarcounterSamples/KitchenSink

**Comment by miyconst** [comment]
@warpech please take a look at the solution @aloscha points to, I found it be good and simple.

**Comment by andwah** [comment]
i took a look, to simple for us to switch to it :-(,  still missing some important features.

On feature is to be able to check if a "form" is valid so a "save" button can be enabled etc..
